src/main/java/uk/gov/justice/digital/service/HiveTableService.java

```diff
@@ -101,6 +105,7 @@ public Set<ImmutablePair<String, String>> switchPrisonsTableDataSource(Immutable
                 String dataPath = createValidatedPath(targetS3Path, sourceName, tableName);
 
                 replaceSymlinkInputTables(jobArguments.getPrisonsDatabase(), hiveTableName, dataPath, schema);
+                storageService.updateDeltaManifestForTable(spark, dataPath);
```

Comment:
This updates the delta manifest so the prisons table is still queryable after it has been switched to use the data located in the temp-reload bucket.